### PR TITLE
Fix building with Xcode 11

### PR DESCRIPTION
**Pull request by s4y**
See commit message for details.

### CODE REVIEW

**Comment by eirnym** [comment]
Thank you for your PR. From my side, I ask only to check where `macvim-askpass` will land for various Xcode versions after your changes to make sure all scripts will work correctly.

**Comment by s4y** [comment]
@eirnym: Could you say more? The `dstSubfolderSpec` change should target the Resources folder for all Xcode versions.

**Comment by eirnym** [comment]
@s4y I don't know how exactly older Xcode versions read this setting. Some people here prefer to keep some compatibility with older systems just because they use them. I don't know how old systems are and which Xcode versions are good to test. Otherwise, I'm fine with your changes.

**Comment by s4y** [comment]
@eirnym Got it; sure. I just went through the Travis build logs for Xcode 7, 8, 9, and 10, which show `macvim-askpass` being copied to Resources, and I verified locally with Xcode 11.

## Travis logs
### xcode7.3
```
PBXCp macvim-askpass build/Release/MacVim.app/Contents/Resources/macvim-askpass
    cd /Users/travis/build/macvim-dev/macvim/src/MacVim
    builtin-copy -exclude .DS_Store -exclude CVS -exclude .svn -exclude .git -exclude .hg -strip-debug-symbols -strip-tool /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/strip -resolve-src-symlinks /Users/travis/build/macvim-dev/macvim/src/MacVim/macvim-askpass /Users/travis/build/macvim-dev/macvim/src/MacVim/build/Release/MacVim.app/Contents/Resources
```

### xcode8.3
```
PBXCp macvim-askpass build/Release/MacVim.app/Contents/Resources/macvim-askpass
    cd /Users/travis/build/macvim-dev/macvim/src/MacVim
    builtin-copy -exclude .DS_Store -exclude CVS -exclude .svn -exclude .git -exclude .hg -strip-debug-symbols -strip-tool /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/strip -resolve-src-symlinks /Users/travis/build/macvim-dev/macvim/src/MacVim/macvim-askpass /Users/travis/build/macvim-dev/macvim/src/MacVim/build/Release/MacVim.app/Contents/Resources
```

### xcode9.4
```
PBXCp macvim-askpass build/Release/MacVim.app/Contents/Resources/macvim-askpass
    cd /Users/travis/build/macvim-dev/macvim/src/MacVim
    builtin-copy -exclude .DS_Store -exclude CVS -exclude .svn -exclude .git -exclude .hg -strip-debug-symbols -strip-tool /Applications/Xcode-9.4.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/strip -resolve-src-symlinks /Users/travis/build/macvim-dev/macvim/src/MacVim/macvim-askpass /Users/travis/build/macvim-dev/macvim/src/MacVim/build/Release/MacVim.app/Contents/Resources
```

### xcode10.2
```
PBXCp /Users/travis/build/macvim-dev/macvim/src/MacVim/macvim-askpass /Users/travis/build/macvim-dev/macvim/src/MacVim/build/Release/MacVim.app/Contents/Resources/macvim-askpass (in target: MacVim)
    cd /Users/travis/build/macvim-dev/macvim/src/MacVim
    builtin-copy -exclude .DS_Store -exclude CVS -exclude .svn -exclude .git -exclude .hg -strip-debug-symbols -strip-tool /Applications/Xcode-10.2.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/strip -resolve-src-symlinks /Users/travis/build/macvim-dev/macvim/src/MacVim/macvim-askpass /Users/travis/build/macvim-dev/macvim/src/MacVim/build/Release/MacVim.app/Contents/Resources
```

File: src/MacVim/MacVim.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -157,7 +157,7 @@
 			isa = PBXCopyFilesBuildPhase;
 			buildActionMask = 2147483647;
 			dstPath = "";
-			dstSubfolderSpec = 6;
+			dstSubfolderSpec = 7;
```
[the remark this answers is not in the document]
Comment (s4y) [reply src/MacVim/MacVim.xcodeproj/project.pbxproj]: Yes! It's a little bit opaque, but this changes when I change the destination in the Xcode project.

File: src/MacVim/gvimrc [hunk 2]
```diff
@@ -37,7 +37,7 @@ endif
 set printexpr=system('open\ -a\ Preview\ '.v:fname_in)\ +\ v:shell_error
 
 " askpass
-let $SSH_ASKPASS = simplify($VIM . '/../../MacOS') . '/macvim-askpass'
+let $SSH_ASKPASS = simplify($VIM . '/../../Resources') . '/macvim-askpass'
```
[the remark this answers is not in the document]
Comment (s4y) [reply src/MacVim/gvimrc]: Sure: I verified locally with Xcode 11, and by reading the Travis build logs for Xcode 7-10. In all four cases it landed in `Resources`.

**Comment by eirnym** [comment]
Thank you

**Comment by markwu** [comment]
The PR works great for me.

**Comment by ychin** [comment]
Thanks for doing this. I double checked in older Xcode versions and the folder seemed fine.

**Pull request merged**